pages/api/login.js

```diff
@@ -50,7 +51,10 @@ async function handler(request, response) {
   }
 
   await client.addEvent(
-    Event.forUser("user_login", UserIdentified.byUserId(user.id))
+    Event.forUser("user_login", UserIdentified.byUserId(user.id)).withMetadata({
+      login_time: new Date(),
+      userId: user.id,
+    })
```

Comment:
```suggestion
    Event.forUser("user_login", UserIdentified.byUserId(user.id))
```

As this data is automatically passed to the API ^^